pallets/identity/src/benchmarking.rs

```diff
@@ -1,16 +1,43 @@
 use super::*;
 
 #[allow(unused)]
-use crate::Pallet as Template;
-use frame_benchmarking::{benchmarks, impl_benchmark_test_suite, whitelisted_caller};
+use crate::Pallet as Identity;
+use frame_benchmarking::{account as benchmark_account, benchmarks, impl_benchmark_test_suite, whitelisted_caller};
 use frame_system::RawOrigin;
+use frame_support::inherent::Vec;
+use codec::alloc::string::{ToString, String};
+
+pub fn from_str_to_vec(string: String) -> Vec<u8> {
+    string.as_bytes().to_vec()
+}
+
+pub fn get_account<T: Config>(name: &'static str) -> T::AccountId {
+    let account: T::AccountId = benchmark_account(name, 0, 0);
```

Comment:
Is this pulling from storage, or calling an extrinsic? Pulling from storage, right?

Reply:
I don't think its either. I am using the account function from `frame_benchmarking::{account as benchmark_account}` to create an account

Reply:
Oh right I misread the import line. Excellent.